redash/handlers/alerts.py

```diff
@@ -28,7 +28,7 @@ def post(self, alert_id):
             'object_id': alert.id,
             'object_type': 'alert'
         })
-
+        models.db.session.commit()
```

Comment:
The commit should be before the event, because if the commit fails we shouldn't log an event. In case you thought the event writes to the DB - the event just sends a task to Celery.